### DISCUSSION TITLE
Can a Android Capacitor plugin written in Kotlin be used by an ionic/capacitor project with Java?

### DISCUSSION THREAD

**Discussion by distante** [ionic-team/capacitor#3632] [Q&A] [answered]
I am really new in Android and what I know until know is mostly based on Kotlin.

I would like to do a capacitor plugin with Kotlin. But I am not sure if by doing so I would prevent people who is using Java to use it.

**Comment by distante** [comment, marked as answer]
I am still searching for an answer for this. If somebody knows about it.

**Reply by pwespi** [reply]
Yes, this should work without any problems.
